policy/modules/kernel/kernel.te

```diff
@@ -259,6 +259,7 @@ sid tcp_socket		gen_context(system_u:object_r:unlabeled_t,mls_systemhigh)
 
 allow kernel_t self:capability2 ~mac_override;
 allow kernel_t self:capability ~sys_module;
+allow kernel_t self:key write;
```

Comment:
I would go one step further and allow the whole `manage_key_perms` set. There is not much benefit in being strict about kernel_t - on the contrary, it might save us some whack-a-mole work later.

Reply:
Done. I also updated the commit message.